### ISSUE TITLE
Make the contact form working

### ISSUE THREAD

**Issue by srijoy-paul** [srijoy-paul/University-website#3]
Feel free to use your preferred language. When the user submits the form after filling it out, all of the information should be sent to a predefined email address.

**Comment by Giriraj-Roy** [comment]
Hello Sir,

I have gone through the repository and the given instructions. The issue seems to attract my interest and excites me to work on it. I humbly wait for you to assign the task to me if your analysis of my profile allows.

Yours Sincerely,
Giriraj Roy.

**Comment by srijoy-paul** [comment]
Sure @Giriraj-Roy you can work on it, assigning you the task

**Comment by srijoy-paul** [comment]
@Giriraj-Roy Are you working on this?

**Comment by Giriraj-Roy** [comment]
I have gone through the sequential process of understanding the working of the repo and especially the [contact.html](https://github.com/srijoy-paul/University-website/blob/main/contact.html) page. 
This is quite clear to me that I will have to add a form-handler.php file that executes the working of the contact form.

I will start applying this process and make a PR within a couple of days.

Thank You.

**Comment by srijoy-paul** [comment]
@Giriraj-Roy Yes, correct, it's not limited to php, you can use any backend language for the same. 
But the task itself is not very time-consuming; you can finish it in a day.

**Comment by srijoy-paul** [comment]
@Giriraj-Roy still working?
Complete within tomorrow 12 noon, else it will be passed to the next person

**Comment by Giriraj-Roy** [comment]
@srijoy-paul I think you need to understand the ecosystem of open source and Hacktoberfest at large.
Try hiring an intern with deadlines.

**Comment by kartiksaxena532** [comment]
hi sir assign me this task i will do it asap and Im really excited to contribute to this.

**Comment by srijoy-paul** [comment]
@kartiksaxena532 Sure go ahead, assigning you the task

**Comment by kartiksaxena532** [comment]
thank you working now on it

**Comment by kartiksaxena532** [comment]
Please merge the pull request @srijoy-paul
